### ISSUE TITLE
Double icon and jumping issue on macOS (Apple Silicon)

### ISSUE THREAD

**Issue by informatik01** [golang/sublime-build#43]
Hi.

After installing this package to Sublime Text 4 and opening Sublime Text, in the macOS's dock a second icon appears and keeps jumping for about a minute or so.

Reinstalling this package didn't help. If I uninstall it - this issue disappears. I.e. it is only present with the "sublime-buld" package installed.

## Related info:
 - MacBook Pro 13'', 2020, Apple M1
 - macOS Ventura, version 13.0.1 (same story was on Monterey version 12.x)
 - Sublime Text, build 4143 (same story was on previous two builds)

**Comment by ucomru** [comment]
Ive this problem too.
Mac OS 13.4 (22F66)
Air 13", 2020, M1

It helped when I set `Preferecnces.sublime-settings` with:
```
"ignored_packages":
	[
		"Golang Build",
	],
```

Process manager shows two Sublime process, one of it not active:

<img width="1078" alt="screen" src="https://github.com/golang/sublime-build/assets/101626426/cae9b409-f53d-4169-840f-b119bb81b2d2">

**Comment by informatik01** [comment]
@ucomru 
Yes, it won't jump anymore, but now this package will be ignored, so you won't be able to "build" any Go program with it.

Try building some Go program (in the menu Tools > Build or with the shortcut <kbd>⌘</kbd> <kbd>B</kbd>) - it won't work anymore, because there will be "No Build System". In other words we can just uninstall this package with the same effect etc.

![image](https://github.com/golang/sublime-build/assets/11339880/fbe4b1b8-4e2c-4a70-9879-f5f14cc963f5)

---
It's really disappointing that the **official** Go package for Sublime Text is not taken care of. Luckily I am using both VS Code text editor and GoLand IDE (for bigger projects). Both provide top-notch experience for Go development.

Anyways, спасибо for the suggestion 🙂

**Comment by RrNn** [comment]
same here

**Comment by RrNn** [comment]
same here, only way to fix it was to ignore the package or just remove it. which makes it un-useful, if i could help i could, but i have no idea how to

**Comment by informatik01** [comment]
This project seems to be **abandoned**: [last commit](https://github.com/golang/sublime-build/commits/master/) was done in July 2022.

So yes, better use something else for Go development. The best options as of now are:
- [Visual Studio Code](https://code.visualstudio.com/) by Microsoft
- [GoLand](https://www.jetbrains.com/go/) by JetBrains (the complete IDE crafted for Gophers)

**Comment by osyu** [comment]
The package causing this is `newterm`; it's only used for the `Go: Open Terminal` palette command, so if you don't mind losing that functionality it can be bodge-fixed by commenting out/removing these lines (as of 71be16f):

- golang_build.py, ln 27 (`import newterm`)
- golang_build.py, ln 361-400 (`class GolangBuildTerminalCommand ...`)
- Default.sublime-commands, ln 14-17 (`... "caption": "Go: Open Terminal" ...`)

This can be done with [OverrideAudit](https://packagecontrol.io/packages/OverrideAudit) by [creating overrides](https://overrideaudit.odatnurd.net/usage/commands/#create-override) for those files.
